CHANGES.rst

```diff
@@ -1,3 +1,425 @@
+Version 1.0rc2.dev21619+gd85adf6ad (2022-04-26)
```

Comment:
What is this? Please undo modification to `CHANGES.rst` and squash this out. Thanks!

Reply:
Oops, looks like I made a mistake with the recent commits.  `CHANGES.rst` reverted.